From 845f9db1e1ccfbfc2dceee22bb7ece1fb167e49f Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:43:12 -0400
Subject: [infiniband] cxgb4: Use uninitialized_var()

Message-id: <1410475447-94392-54-git-send-email-svemuri@redhat.com>
Patchwork-id: 93806
O-Subject: [RHEL7.1 PATCH BZ 1124947 053/108] RDMA/cxgb4: Use uninitialized_var()
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Steve Wise <swise@opengridcomputing.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 97df1c6736f660b58b408a60d0f7f65a64fb9d56)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/cxgb4/cq.c b/drivers/infiniband/hw/cxgb4/cq.c
index e17b155..cfaa56a 100644
--- a/drivers/infiniband/hw/cxgb4/cq.c
+++ b/drivers/infiniband/hw/cxgb4/cq.c
@@ -672,7 +672,7 @@ skip_cqe:
 static int c4iw_poll_cq_one(struct c4iw_cq *chp, struct ib_wc *wc)
 {
  struct c4iw_qp *qhp = NULL;
- struct t4_cqe cqe = {0, 0}, *rd_cqe;
+ struct t4_cqe uninitialized_var(cqe), *rd_cqe;
  struct t4_wq *wq;
  u32 credit = 0;
  u8 cqe_flushed;
-- 
1.7.1